### ISSUE TITLE
Hiding ActionBars via unchecking "enable" option now completely prevents ActionBar from functioning.

### ISSUE THREAD

**Issue by ghensai** [tukui-org/ElvUI#1102]
### Troubleshooting Steps

![Screenshot 2023-11-17 192254](https://github.com/tukui-org/ElvUI/assets/114158969/45fde367-4cb9-46f4-bd8d-d80376c0d4b3)

As the issue relates to a keybind not functioning when actions bars are disabled, it is physically impossible for me to SHOW the issue in a screenshot.

### Describe the expected behavior and what actually happened?

For the last ~10 years I have used ElvUI, I have hidden my ActionBars simply by unchecking the "enable" option for each bar.

I still USE the ActionBars, they are full of abilities/keybinds, I just don't want to SEE them. So I hide them. Unchecking "enable" has worked for this for as long as I have used ElvUI.

I was playing the game today just fine, then as I was about to enter a dungeon I updated to 13.52. Immediately my keybinds stopped working. Eventually I figured out that if I check "enable" they start working again, and as soon as I uncheck "enable" (returning to the state in which I have used ElvUI for ~10 years), they stopped working.

Obviously, something has changed in 13.52 that has broken the way this functioned previously. Perhaps it is intended that when a bar is not "enabled" it simply does not function. However, this is NOT how this has worked until now, and completely breaks how I use ElvUI. I do not want to see my action bars. I want them populated and bound, but invisible. If this is intended functionality, please add an option to "hide" ActionBars.

### Reproducing the issue.

1. Enable ActionBar1.
2. Add an ability to ActionBar1.
3. Keybind the ability added to ActionBar1 in Step 2.
4. Uncheck "enable" for ActionBar1.
5. Press the keybind assigned in Step 3.
6. Ability will not work.

If you use the version of ElvUI prior to 13.52 and repeat the above steps, the keybind will function normally.

### Lua Errors

```shell
None.
```

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/development/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by 1Botanica1** [comment]
Yes, there has been a change to improve the performance of ElvUI
The bars that are disabled now cannot be used - they are properly disabled. 

There is already an option to put hide in the action bar visibility state

• Actionbars which were disabled but used for keybinds, now need to be enabled and visibility state changed to hide to work like before

From the changelog

Duplicate of #1095
